lib/pbench/test/unit/common/test_wait_for_uri.py

```diff
@@ -33,6 +33,35 @@ def test_wait_for_uri_bad():
 
 
 def setup_conn_ref(monkeypatch):
+    """Setup a mock'd up environment for socket.create_connection to drive
```

Comment:
OK, fine, but why not just "mocked" instead of "mock'd [up]"? 🍤

Reply:
I'm not sure that I can explain or justify it, but "mock'd" is simply _right_.  🤣 

("Mocked" is what you are when people make fun of you.  😉)

Reply:
One way is just bad spelling, the other is bad behavior ... but whatever.

Reply:
https://tenor.com/view/thor-are-you-mocking-me-chris-hemsworth-inifnity-war-avengers-gif-11778832

Reply:
Sure, let's just be silly; but "mock'd" will continue to really bug me. Ah, well; there are worse demons and monsters loose in the world.